package.json

```diff
@@ -0,0 +1,45 @@
+{
+  "name": "design-system",
+  "version": "1.0.0",
+  "description": "Design system of MB",
+  "main": "index.js",
+  "directories": {
+    "doc": "docs"
+  },
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "storybook": "start-storybook -p 6006",
+    "build-storybook": "build-storybook"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/chhavijustme/design-system.git"
+  },
+  "keywords": [
+    "design-system"
+  ],
+  "author": "",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/chhavijustme/design-system/issues"
+  },
+  "homepage": "https://github.com/chhavijustme/design-system#readme",
+  "dependencies": {
+    "css-loader": "^0.28.11",
+    "less": "^3.0.2",
+    "less-loader": "^4.1.0",
+    "npm": "^6.0.0",
```

Comment:
`npm` can be removed from package.json. I would actually prefer if we used [Yarn](https://yarnpkg.com/en/) for package management, since npm has caused us a lot of pain in the past.